dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java

```diff
@@ -322,6 +322,22 @@ public List<Entry> findAllGitHubEntriesWithNoTopicAutomatic() {
         return list(this.currentSession().getNamedQuery("Entry.findAllGitHubEntriesWithNoTopicAutomatic"));
     }
 
+    public List<Long> getWorkflowIds(int offset, int pageSize,
```

Comment:
A little confused, are these two methods used anywhere and do these two named queries exist? 
Possible leftovers?

Reply:
Shoot, this was my initial attempt to try to streamline the operation by not fetching workflows, just their ids, which I then forgot to rollback.